### PR TITLE
Have a "common errors" page.

### PR DESCRIPTION
**Pull request by kflynn**
This is obviously not a complete list -- it's meant to be a start that we add to.

Signed-off-by: Flynn <flynn@buoyant.io>

### CODE REVIEW

File: linkerd.io/content/common-errors/_index.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+      is sane, and it will point you to help for any problems it finds. It's
+      hard to overstate how useful this command is.
+    answer_schema: |-
+      Always start with linkerd check. Whenever you see anything that looks
+      unusual about your mesh, always start with linkerd check. It will check
+      a long series of things that have caused trouble for others and make
+      sure that your configuration is sane, and it will point you to help
+      for any problems it finds. It's hard to overstate how useful this
+      command is.
+  - question: Why am I seeing protocol detection timeouts?
```
Comment (wmorgan) [inline linkerd.io/content/common-errors/_index.md:28]: I think this needs a much more in-depth answer

**Review by wmorgan — COMMENTED** [review]
A good start, thanks. I think this whole schema thing is a huge maintenance burden with very dubious value and I would argue for making this just a regular page.

File: linkerd.io/content/common-errors/_index.md (hunk 1 above)
Comment (wmorgan) [reply linkerd.io/content/common-errors/_index.md:28]: I.e. the answer isn't just to blindly mark these things with skip or opaque ports, which have their own set of implications. And these errors can be transient... basically you have to understand a lot more about the situation before you can decide whether you want to even change anything.

**Review by alpeb — COMMENTED** [review]
In order to avoid having these links pointing to 2.15 staling out on each major release, I suggest moving this doc to `linkerd.io/content/2.15/common-errors/_index.md`, and use relative links instead. The `common-errors` doc will get copied then on each major release, like we do with the rest, without further changes required.

File: linkerd.io/content/common-errors/_index.md (hunk 1 above)
Comment (kflynn) [reply linkerd.io/content/common-errors/_index.md]: See updates. 🙂

File: linkerd.io/layouts/partials/sidebar-2.html [hunk 2]
```diff
@@ -96,6 +96,11 @@
           Frequently Asked Questions
         </a>
       </li>
+      <li class="menu-list-item {{ if eq $currentPage.RelPermalink "/common-errors/" }}is-active{{ end }}">
+        <a href="/common-errors/">
```
[the remark this answers is not in the document]
Comment (kflynn) [reply linkerd.io/layouts/partials/sidebar-2.html:100]: I think I can just use pure relative links now, actually...

**Comment by kflynn** [comment]
I moved this into a normal section, parallel to `features` -- better?

**Review by wmorgan — APPROVED** [review]

**Pull request merged**